modules/nf-core/metaphlan/makedb/main.nf

```diff
@@ -6,9 +6,11 @@ process METAPHLAN_MAKEDB {
         'https://depot.galaxyproject.org/singularity/metaphlan:4.0.6--pyhca03a8a_0' :
         'biocontainers/metaphlan:4.0.6--pyhca03a8a_0' }"
```

Comment:
Can you update the version to the latest: 4.1.1?

Reply:
Thank you @LilyAnderssonLee 

That's done now!